qa/tasks/keystone.py

```diff
@@ -113,6 +113,40 @@ def download(ctx, config):
                 args=[ 'rm', '-rf', keystonedir ],
             )
 
+@contextlib.contextmanager
+def install_packages(ctx, config):
```

Comment:
@shyukri could you drop `install_packages()` from this source file. as mimic does not have `install_packages()` by then.